### PR TITLE
Rename do not switch case

### PR DESCRIPTION
**Pull request by fabianfett**

### CODE REVIEW

File: Sources/Tracing/SpanProtocol.swift [hunk 1]
```diff
@@ -327,7 +327,7 @@ public enum SpanAttribute: Equatable {
     case string(String)
     case stringArray([String])
 
-    case __DO_NOT_SWITCH_EXHAUSTIVELY_OVER_THIS_ENUM
+    case __DO_NOT_SWITCH_EXHAUSTIVELY_OVER_THIS_ENUM_USE_DEFAULT_INSTEAD
```
Comment (FranzBusch) [inline Sources/Tracing/SpanProtocol.swift:330]: Maybe we can add some written comment to this case as well clarifying this a bit more?

**Review by FranzBusch — APPROVED** [review]
We can add the comment in a follow up

File: Sources/Tracing/SpanProtocol.swift (hunk 1 above)
Comment (ktoso) [reply Sources/Tracing/SpanProtocol.swift:330]: We'll add in a follow up

Comment (ktoso) [reply Sources/Tracing/SpanProtocol.swift:330]: We'll add in a follow up

**Review by ktoso — APPROVED** [review]

**Pull request merged**